src/cltk/dependency/spacy_wrapper.py

```diff
@@ -15,14 +15,17 @@
 LOG.addHandler(logging.NullHandler())
 
 MAP_LANGS_CLTK_SPACY = {
+    "grc": "grc",
     "lat": "la",
 }
 
 MAP_LANG_TO_SPACY_MODEL_NAME: dict[str, str] = {
+    "grc": "grc_odycy_joint_sm",
     "lat": "la_core_web_lg",
 }
 
 MAP_LANG_TO_SPACY_MODEL_URL: dict[str, str] = {
+    "grc": "https://huggingface.co/chcaa/grc_odycy_joint_sm/resolve/main/grc_odycy_joint_sm-any-py3-none-any.whl",
```

Comment:
I'm surprised that they call their one model "sm" but there is no medium or large.

Reply:
They actually have this model which looks to have better results: https://huggingface.co/chcaa/grc_odycy_joint_trf, https://huggingface.co/chcaa/grc_odycy_joint_trf/resolve/main/grc_odycy_joint_trf-any-py3-none-any.whl. Do we prefer a smaller model with good results or a heavier results with the best results?